db/migrate/20170508085402_add_not_null_constraint_to_sources_payment_type.rb

```diff
@@ -0,0 +1,11 @@
+class AddNotNullConstraintToSourcesPaymentType < ActiveRecord::Migration
+  def change
+    reversible do |dir|
+      dir.up do
+        SolidusPaypalBraintree::Source.where(payment_type: nil).
+          update_all(payment_type: 'CreditCard')
```

Comment:
This isn't accurate. Is there a scenario that generated nil payment_types in the past?